edx-platform/pearson-ukl-theme/lms/templates/header/navbar-not-authenticated.html

```diff
@@ -50,7 +49,7 @@
   <div class="secondary">
     <div>
       % if allows_login:
-        % if allow_public_account_creation:
+        % if not static.get_value('HIDE_REGISTRATION', False):
```

Comment:
It is possible to declare this as a variable at the beginning of the file as it has been done with the others? @Squirrel18

Reply:
There is no need to create a variable that is only used once.